src/analytics.ts

```diff
@@ -139,13 +139,8 @@ export class Analytics extends Emitter {
   }
 
   async page(...args: PageParams): Promise<DispatchedEvent> {
-    const [
-      category,
-      page,
-      properties,
-      options,
-      callback,
-    ] = resolvePageArguments(...args)
+    const [category, page, properties, options, callback] =
```

Comment:
What rule changed this behavior? Purely subjective tastes - I find the former easier to read compared to an `=` followed by newline but that might just be me.

Reply:
I agree, but this was something `new` in prettier, which I assume was basically choosing to not wrap list items like this unless it hit the print width.

Reply:
I just checked and `prettier-eslint-config` (correctly) disables it, as it conflicts with prettier's opinions. I think we can enable something like that by adding a rule for  "array-element-newline".